setup/ci/codebuild.md

```diff
@@ -18,7 +18,26 @@ The AWS Codebuild stage requires Spinnaker 1.19 or later.
 
 ### AWS CodeBuild project
 
-You need to have an [AWS CodeBuild](https://aws.amazon.com/codebuild/) project.
+You need to have an [AWS CodeBuild](https://aws.amazon.com/codebuild/) project. To create a project,
+follow the instructions on how [to create a CodeBuild service role (AWS CLI)](https://docs.aws.amazon.com/codebuild/latest/userguide/setting-up.html#setting-up-service-role-cli).
+
+Next, run the following command (make sure you replace the service role with the one created and region of your own choice):
+
+```
+aws codebuild create-project \
+  --name spinnaker-project \
+  --source "type=GITHUB,location=https://github.com/aws-samples/aws-codebuild-samples.git" \
```

Comment:
Ooh actually I just realized that project hasn't been updated in a while with the newer runtime-versions options.  Can you update that repo and make sure it actually builds successfully with this project configuration?

Reply:
This repo could be successfully built with the newer standard image because we make the runtime-versions switch backwards compatible. I've run a build to double check and it succeeded.

Reply:
Perfect, thanks!